rating.html

```diff
@@ -47,7 +47,8 @@
     <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bulma@0.8.0/css/bulma.min.css">
     <script src="app.js"></script>
 </head>
- <body>
+
+ <body onload="returnMovie()">>
```

Comment:
You may want to consider specifying the "onload" trigger in JavaScript. To do this we would setup separate JS files for each page, which may also help with merge conflicts. What are your thoughts?

Reply:
I'm a bit confused as to how this would work so we should have a call to go over it.  Regardless, lets wait until the next sprint to split the JS file because that will be confusing for the team this late in this current sprint.